go/cmd/vtworker/vtworker.go

```diff
@@ -78,9 +78,11 @@ func main() {
 		go func() {
 			if err := wi.WaitForCommand(worker, done); err != nil {
 				log.Error(err)
+				log.Flush()
```

Comment:
It should be safe to replace log.Flush() with logutil.Flush(). That package wraps all the differences.